cloud_infrastructure_access/rosa-aws-privatelink.adoc

```diff
@@ -0,0 +1,17 @@
+[id="rosa-aws-privatelink"]
+= Configuring AWS PrivateLink
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: rosa-aws-privatelink
+
+toc::[]
+
+include::modules/rosa-configure-privatelink-about.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-architecture.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-requirements.adoc[leveloffset=+1]
+include::modules/rosa-configure-privatelink-configuration.adoc[leveloffset=+1]
+
+== Additional resources
+
+* See xref:../rosa_getting_started/rosa-setting-up-environment.adoc[Setting up the environment] to prepare for installing {product-title} (ROSA).
```

Comment:
s/to prepare for installing {product-title} (ROSA)/to prepare your environment and install {product-title} (ROSA)